pkgs/by-name/ll/lldap/package.nix

```diff
@@ -49,7 +88,20 @@ let
 
       installPhase = ''
         mkdir -p $out
-        cp -R app/{index.html,pkg,static} $out/
+        cp -R app/{pkg,static} $out/
+        cp app/index_local.html $out/index.html
+
+        rm $out/static/libraries.txt $out/static/fonts/fonts.txt
```

Comment:
The easiest way would be to just put an assert on the sha sum of these files, instead of just deleting them.